website/docs/sdk-reference/elixir.md

```diff
@@ -0,0 +1,235 @@
+
+### Auto polling (default)
+
+The *ConfigCat SDK* downloads the latest values and stores them automatically every 60 seconds.
+
+Use the `poll_interval_seconds` option parameter to change the polling interval.
+```elixir
+{ConfigCat, [
+    sdk_key: "YOUR SDK KEY",
+    cache_policy: CachePolicy.auto(poll_interval_seconds: 60)
+]},
+```
+Adding a callback to `on_changed` option parameter will get you notified about changes.
+```elixir
+{ConfigCat, [
+    sdk_key: "YOUR SDK KEY",
+    cache_policy: CachePolicy.auto(on_changed: callback/0)
+]}
+```
+
+Available options:
+
+| Option Parameter                    | Description                                                                                          | Default |
+| ----------------------------------- | ---------------------------------------------------------------------------------------------------- | ------- |
+| `poll_interval_seconds`             | Polling interval.                                                                                    | 60      |
+| `on_changed` | A 0-arity function to be called about configuration changes. Nees to run on a separate proccess. Any exceptions raised by `on_changed` are caught and logged. | -       |
+
+### Lazy loading
+
+When calling `get_value()` the *ConfigCat SDK* downloads the latest setting values if they are not present or expired in the cache. In this case the `get_value()` will return the setting value after the cache is updated.
+
+Use `cache_expiry_seconds` option parameter to set cache lifetime.
+```elixir
+{ConfigCat, [
+    sdk_key: "YOUR SDK KEY",
+    cache_policy: CachePolicy.lazy(cache_expiry_seconds: 300)
+]}
+```
+
+Available options:
+
+| Option Parameter             | Description                                                                                   | Default |
+| ---------------------------- | --------------------------------------------------------------------------------------------- | ------- |
+| `cache_expiry_seconds` | Cache TTL.                                                                                    | 60      |
+
+### Manual polling
+
+Manual polling gives you full control over when the setting values are downloaded. *ConfigCat SDK* will not update them automatically. Calling `force_refresh()` is your application's responsibility.
+
+```elixir
+ConfigCat.force_refresh();
+```
+
+> `get_value()` returns `default_value` if the cache is empty. Call `force_refresh()` to update the cache.
+```elixir
+value = ConfigCat.get_value("key", "my default value") # Returns "my default value"
+ConfigCat.force_refresh();
+value = ConfigCat.get_value("key", "my default value") # Returns "value from server"
+```
+
+### Custom cache behaviour with `cache:` option parameter.
```

Comment:
Please remove the ending ".".

Reply:
Please refresh the page, I think you might be seeing an older version of this PR